Library/Homebrew/test/support/helper/env.rb

```diff
@@ -1,14 +1,20 @@
 module Test
   module Helper
     module Env
+      def copy_env
+        ENV.to_hash
+      end
+
+      def restore_env(env)
+        ENV.replace(env)
+      end
+
       def with_environment(partial_env)
```

Comment:
(Alternatively, I could get rid of all of Homebrew's uses of it and move it into cask.)